### ISSUE TITLE
Unable to silence generic type arg count errors

### ISSUE THREAD

**Issue by smallnamespace** [python/mypy#1404]
I expect to be able to silence errors relating to number of type args in generic types, but I can't.

This piece of code
```python
from typing import Iterable, no_type_check


@no_type_check
class Foo(Iterable[int, float]):  # type: ignore
    pass
```

results in

```mypy.py:5: error: "Iterable" expects 1 type argument, but 2 given```

even though I've used both ```@no_type_check``` and ```# type: ignore```.

Note that I'm purposefully passing in the wrong number of generic args (to a custom generic subclass, obviously, not to `Iterable`) in order to work around https://github.com/python/mypy/issues/1304, since the `typing.py` fix hasn't filtered out yet.

**Comment by JukkaL** [comment]
Also, `Iterable[int, float]` should probably be understood as `Iterable[Any]` internally.

**Comment by ddfisher** [comment]
Looks like this was fixed by e6ba756d8a8337a2c5337864bb028faaa4123623.
